jvb/rootfs/defaults/jvb.conf

```diff
@@ -45,11 +45,12 @@ videobridge {
     }
     http-servers {
         private { 
-          host = 0.0.0.0
+          host = "{{ .Env.JVB_HTTP_SERVER_PRIVATE_ADDRESS | default "0.0.0.0" }}"
```

Comment:
I don't think this needs quotation marks.

Reply:
I copied the style from the other config options:
https://github.com/jitsi/docker-jitsi-meet/blob/master/jvb/rootfs/defaults/jvb.conf#L59

But I'll remove the quotation marks.